pkg/pipeline/write/write_tcp.go

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2021 IBM, Inc.
```

Comment:
Fix (or remove) Copyright statement

Reply:
This is coming from both [eBPF packets exporter](https://github.com/netobserv/netobserv-ebpf-agent/blob/9507ebf9fbc5ba65f40befa4a55011e92ef32068/pkg/exporter/packets.go) and [flp stdout write stage](https://github.com/netobserv/flowlogs-pipeline/blob/main/pkg/pipeline/write/write_stdout.go) :smile: 

I just copied the code from there but I can remove the copyright if you want.